src/Components/App/App.js

```diff
@@ -90,7 +83,10 @@ const App = () => {
   const resestResults = (event) => {
     setError('')
     event.preventDefault()
-    setVendors(vendors)
+    console.log(vendors, 'all vendors after reset')
+    console.log(storedVendors, 'stroed vendosr reset')
+    const correctVendors = vendors.length === 10 ? vendors : storedVendors
```

Comment:
Glad this worked, thanks for working through it. I will have to see it in action and hear your thought process of why it was necessary

Reply:
Interesting fix!!